src/ui/ConnectToServer/ConnectToServer.scss

```diff
@@ -70,7 +70,7 @@
     top: -$spacer;
 
     &__Btn {
-      margin: 0 ($spacer / 4);
+      margin: 0 ($spacer * 0.25);
```

Comment:
Interestingly enough, this is what the sass migrator tool does for this: https://github.com/sass/migrator#readme

@kraenhansen It is a lot less verbose now, what do you think?